sdk/appconfiguration/app-configuration/package.json

```diff
@@ -52,13 +52,17 @@
     "prebuild": "npm run clean",
     "pack": "npm pack 2>&1",
     "swagger": "autorest --typescript swagger/swagger.md",
-    "test": "npm run build:test && mocha -t 1200000 test-dist/index.node.js --reporter ../../../common/tools/mocha-multi-reporter.js",
-    "unit-test:browser": "echo skipped",
-    "unit-test:node": "npm run test",
-    "unit-test": "npm run unit-test:node && npm run unit-test:browser",
     "integration-test:browser": "echo skipped",
-    "integration-test:node": "npm run test",
-    "integration-test": "npm run unit-test:node && npm run unit-test:browser"
+    "integration-test:node": "nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 180000 --full-trace dist-esm/test/*.spec.js dist-esm/test/**/*.spec.js",
+    "integration-test:node:no-timeout": "nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --no-timeouts --full-trace dist-esm/test/*.spec.js dist-esm/test/**/*.spec.js",
+    "integration-test": "npm run unit-test:node && npm run unit-test:browser",
+    "test:browser": "npm run clean && npm run build:test && npm run unit-test:browser",
+    "test:node": "npm run clean && npm run build:test && npm run unit-test:node",
+    "test": "npm run clean && npm run build:test && npm run unit-test",
+    "unit-test:browser": "echo skipped",
+    "unit-test:node": "mocha --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 180000 --full-trace dist-test/index.node.js",
+    "unit-test:node:no-timeout": "mocha --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --no-timeouts --full-trace dist-test/index.node.js",
```

Comment:
When would you suggest using the version with timeouts versus the one without timeouts?

Reply:
The version with timeouts is necessary for CI. The one without timeouts can be useful for debugging! I copied that one from keyvault. Should I remove the one without timeouts?

Reply:
I don't feel strongly either way, I just wanted to know why it existed since I hadn't seen that before :smile:

Reply:
--no-timeouts lets you run the test indefinitely?

Reply:
Yep! You can see it somewhere in this section: https://mochajs.org/#-timeout-ms-t-ms

Reply:
I'd remove it - it's not needed for AppConfig and I'd like to not encourage it :)

Reply:
Removing!

Reply:
`unit-test:node:no-timeout` seems like it really shouldn't exist. If they're actually unit-tests they shouldn't time out at all, right?